cget/builder.py

```diff
@@ -87,4 +87,5 @@ def test(self, variant=None):
         if 'check' in self.targets():
             self.build(target='check', variant=variant or 'Release')
         else:
-            self.prefix.cmd.ctest((self.prefix.verbose and ['-VV'] or []) + ['-C', variant], cwd=self.build_dir)
+            self.prefix.cmd.ctest((self.prefix.verbose and ['-VV'] or []) + ['-C', variant] +
```

Comment:
Do we need `-VV` if we have `--output-on-failure`? It seems this could be removed.

Reply:
I think it is still useful to keep -VV, because that allows you to see all test output even if there are no failures.

Remember, this is output from the actual tests that run - by default you don't see any of that regardless of whether there are failures.